modules/pubxaiAnalyticsAdapter.js

```diff
@@ -92,6 +92,7 @@ function mapBidResponse(bidResponse, status) {
       Object.assign(bid, {
         bidId: bidResponse.requestId,
         floorProvider: events.floorDetail ? events.floorDetail.floorProvider : null,
+        floorExtraData: events.floorDetail ? events.floorDetail : null,
```

Comment:
Could you be explicit in what data you are picking from `floorDetail`? the problem with just including it raw in your payload is that it forces Prebid to never change its structure from now on, regardless of what you are actually using from it in your backend.

Reply:
@dgirardi Yeah makes sense. I have updated it to pick only the required fields. Thanks!